packages/terra-form/src/components/Control.jsx

```diff
@@ -0,0 +1,73 @@
+// eslint-disable react/jsx-boolean-value
+
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+
+import Input from './Input';
+
+const propTypes = {
+  /**
+   * The input type
+   */
+  type: PropTypes.oneOf(['checkbox', 'radio']).isRequired,
+  /**
+   * Additional attributes for the input object
+   */
+  attrs: PropTypes.object,
```

Comment:
`inputAttributes` seems extremely verbose.... FWIW, this follows the pattern of what Django does which uses `attrs` and has worked there. I'd rather have to check the documentation once, learn what it is and then use that going forward over having to type `inputAttributes` every time I set this... 

https://docs.djangoproject.com/en/1.11/ref/forms/widgets/#django.forms.Widget.attrs